src/lib/RequestTypes.ts

```diff
@@ -328,10 +338,11 @@ export class AccountsRequest {
                     ),
                 } as ParsedCheckoutRequest;
             case RequestType.ONBOARD:
-                request = request as SimpleRequest;
+                request = request as OnboardingRequest;
                 return {
                     kind: RequestType.ONBOARD,
                     appName: request.appName,
+                    hideBack: request.hideBack === true,
```

Comment:
Another possibility (not saying it's better, just possible :wink:)
```suggestion
                    hideBack: !!request.hideBack,
```

Reply:
Never liked that. No no no :D

Reply:
Especially, in this case I want that value to be exactly true, and not "nipir32pnp132r" or whatever would be possible with `!!` ;)